app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -137,13 +138,18 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
     private LeanbackOverlayFragment leanbackOverlayFragment;
 
     // Subtitle fields
-    private static final int SUBTITLE_PADDING = 8;
+    private static final int SUBTITLE_PADDING = 20;
     private static final long SUBTITLE_RENDER_INTERVAL_MS = 50;
     private SubtitleTrackInfo subtitleTrackInfo;
     private int currentSubtitleIndex = 0;
-    private int subtitlesSize = KoinJavaComponent.<UserPreferences>get(UserPreferences.class).get(UserPreferences.Companion.getDefaultSubtitlesSize());
+    private final int subtitlesSize = KoinJavaComponent.<UserPreferences>get(UserPreferences.class).get(UserPreferences.Companion.getDefaultSubtitlesSize());
```

Comment:
Inject UserPreferences once instead of for each property

Reply:
Now done, i really appreciate the help here, thanks for the assistance, hoping the next PR i do is a lot less work for you